docs/blogs/conformance.html

```diff
@@ -0,0 +1,156 @@
+        <pre>
+            <code class="language-rust">
+use commonware_codec::{conformance::CodecConformance, Write, FixedSize};
+use bytes::BufMut;
+use commonware_conformance::conformance_tests;
+
+#[derive(arbitrary::Arbitrary))]
+struct MyType([u8; Self::SIZE]);
+
+impl Write for MyType {
+    fn write(&amp;self, buf: &amp;mut impl BufMut) {
+        buf.put_slice(&amp;self.0);
+    }
+}
+
+impl FixedSize for MyType {
+    const SIZE: usize = 8;
+}
+
+conformance_tests! {
+    // Generate 1024 test cases for MyType using seeds 0..1024
+    CodecConformance&lt;MyType&gt; =&gt; 1024
+}
+            </code>
+        </pre>
+        This test will generate a <code>conformace.toml</code> file in the crate root with the following contents:
+        <pre>
+            <code class="language-toml">
+["crate::CodecConformance&lt;MyType&gt;"]
+n_cases = 1024
+hash = "dd8bf4b8ca06978d565c0aef2c2b8823c7b3fdcb065a4d1c1b33fe76597c52c6"
+            </code>
+        </pre>
+        <p>Any time we change the code, we can run the conformance tests to ensure that the generated commitments match what is expected. If they don't, we know that we've introduced a breaking change and can take steps to explicitly acknowledge or avoid it.</p>
+        <p><code>conformance</code> is also useful for validating the semantics of mechanisms. For example, we use this framework to <a href="https://github.com/commonwarexyz/monorepo/blob/66c2f637029774652e976b10e81449d0c17a863f/cryptography/src/handshake/conformance.rs">assert the stability of <code>cryptography</code>'s key exchange protocol</a>, as well as <a href="https://github.com/commonwarexyz/monorepo/blob/66c2f637029774652e976b10e81449d0c17a863f/storage/src/journal/conformance.rs#L1-L1">the consistency of on-disk contents in <code>storage</code>'s fixed and variable <code>Journal</code>s</a>.</p>
+
+        <h2>Commonware's Commitment to Stability</h2>
+        <p>Commonware is coming up on its first stable release, and we are committed to providing stable and reliable software for our users. By adopting conformance testing as a standard practice, we can ensure that our libraries' functionality remains robust and trustworthy, even as we continue to innovate on and evolve the underlying mechanisms.</p>
+        <p>In our codebase today, <code>conformance</code> covers:</p>
+        <ul>
+            <li>Wire formats (<code>codec</code>)</li>
+            <li>Database formats (<code>storage</code>)</li>
```

Comment:
nit: I think we should use a different word than `format` for this (it also includes how many things may be written/in what order/etc.)